components/micro-gateway-core/src/main/java/org/wso2/micro/gateway/core/jwt/generator/MGWJWTGeneratorImpl.java

```diff
@@ -0,0 +1,126 @@
+/*
+ *  Copyright (c) 2020, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *  http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ */
+
+package org.wso2.micro.gateway.core.jwt.generator;
+
+import org.apache.commons.lang3.StringUtils;
+import org.apache.commons.logging.Log;
+import org.apache.commons.logging.LogFactory;
+
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.UUID;
+
+/**
+ *  Class to implement standard claims and custom claims.
+ */
+public class MGWJWTGeneratorImpl extends AbstractMGWJWTGenerator {
+    private static final Log logger = LogFactory.getLog(MGWJWTGeneratorImpl.class);
+    private static final String AUTH_APPLICATION_USER_LEVEL_TOKEN = "Application_user";
+
+    public MGWJWTGeneratorImpl(String dialectURI,
+                               String signatureAlgorithm,
+                               String trustStorePath,
+                               String trustStorePassword,
+                               String certificateAlias,
+                               String privateKeyAlias,
+                               int jwtExpiryTime,
+                               String[] restrictedClaims,
+                               boolean jwtCacheEnabled,
+                               int jwtCacheExpiry,
+                               String tokenIssuer,
+                               String[] tokenAudience) {
+        super(dialectURI, signatureAlgorithm, trustStorePath, trustStorePassword, certificateAlias, privateKeyAlias,
+                jwtExpiryTime, restrictedClaims, jwtCacheEnabled, jwtCacheExpiry, tokenIssuer, tokenAudience);
+    }
+
+    @Override
+    public Map<String, Object> populateStandardClaims(Map<String, Object> jwtInfo) {
+        long currentTime = System.currentTimeMillis();
+        long expireIn = currentTime + getTTL();
+        String dialect = this.getDialectURI();
+        Map<String, Object> claims = new HashMap<>();
+        HashMap<String, Object> customClaims = (HashMap<String, Object>) jwtInfo.get("customClaims");
```

Comment:
I think we can put the common strings used here in a constant file and use those constants instead. If they are already defined as constants let's use that. Ex: "customClaims", "sub", "iss", "tier" etc.

Reply:
Done with https://github.com/wso2/product-microgateway/pull/1613/commits/45bcd3462390fb8a500155b1accc2e40b6f75444